templates/default/postfix-main.cf.erb

```diff
@@ -18,20 +11,21 @@ append_dot_mydomain = no
 readme_directory = no
 
 # TLS parameters
-smtpd_tls_cert_file=/etc/ssl/certs/ssl-cert-snakeoil.pem
-smtpd_tls_key_file=/etc/ssl/private/ssl-cert-snakeoil.key
+smtpd_tls_cert_file=<%= @postfix_cert_path %>
 smtpd_use_tls=yes
-smtpd_tls_session_cache_database = btree:/smtpd_scache
-smtp_tls_session_cache_database = btree:/smtp_scache
+smtpd_tls_loglevel = 1
+smtpd_tls_session_cache_database = btree:${data_directory}/smtpd_scache
+smtp_tls_session_cache_database = btree:${data_directory}/smtp_scache
```

Comment:
The original config wasn't broken but it was creating some warnings in the maillog like
```
Jan 25 11:08:00 admin1 postfix/tlsmgr[3689]: warning: request to update table btree:/smtpd_scache in non-postfix directory /
Jan 25 11:08:00 admin1 postfix/tlsmgr[3689]: warning: redirecting the request to postfix-owned data_directory /var/lib/postfix
```